presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java

```diff
@@ -13,27 +13,66 @@
  */
 package io.prestosql.server.rpm;
 
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
 import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
 
 import java.sql.Connection;
 import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.HashSet;
 import java.util.Set;
 
+import static java.lang.String.format;
 import static java.sql.DriverManager.getConnection;
 import static java.util.Arrays.asList;
+import static org.testcontainers.containers.wait.strategy.Wait.forLogMessage;
 import static org.testng.Assert.assertEquals;
 
 public class ServerIT
 {
-    @Parameters("presto.port")
+    @Parameters({"target", "rpm"})
     @Test
-    public void testServer(int port)
+    public void testServer(String target, String rpm)
             throws Exception
     {
-        String url = "jdbc:presto://localhost:" + port;
+        testServer("prestodev/centos7-oj8", target, rpm);
+    }
+
+    public void testServer(String image, String target, String rpm)
+            throws Exception
+    {
+        String command = "" +
+                // install RPM
+                "rpm -i /target/" + rpm + "\n" +
```

Comment:
if you bind rpm as a single file, this will be sth like
```
rpm -i /presto-server.rpm
```

moreover, this should be 
```
yum localinstall -y the.rpm
```

Reply:
Done